contract/models/contract.py

```diff
@@ -428,6 +428,7 @@ def _prepare_invoice(self, date_invoice, journal=None):
             'journal_id': journal.id,
             'origin': self.name,
             'user_id': self.user_id.id,
+            'old_contract_id': self.id
```

Comment:
Please don't use this deprecated field. You need to inject the followers on the created invoices guessing them from the computed field, the same as the smart-button does.